terraform/environments/core-shared-services/instance-scheduler-lambda-function.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_lambda_function" "instance-scheduler-lambda-function" {
+  #checkov:skip=CKV_AWS_116
+  #checkov:skip=CKV_AWS_117
+  #checkov:skip=CKV_AWS_272 "Code signing not required"
+  function_name                  = "instance-scheduler-lambda-function"
+  handler                        = "main"
+  reserved_concurrent_executions = 0
+  runtime                        = "go1.x"
+  image_uri                      = "${module.instance_scheduler_ecr_repo.ecr_repository_name}:latest"
```

Comment:
latest tag should be default, but ideally we want to pass this as a variable, don't we?

Reply:
We use terraform variables to pass different values to terraform modules. In this case, the lambda is not part of a module and only deployed as a single instance in core-shared-services. I don't see why we would be using a variable here, but let me know if you could point me to an example in the existing code.

Reply:
cool ,thanks for the explanation

Reply:
ok, let's tackle it in the future